bootkube/resources/charts/calico.yaml

```diff
@@ -9,3 +9,27 @@ calico:
   ipipReadiness: "${ipip_readiness}"
   podCIDR: ${pod_cidr}
   networkEncapsulation: "${network_encapsulation}"
+  hostProtection:
+    enabled: ${host_protection}
+    %{~ if node_cidr != "" ~}
+    nodeCIDR: ${node_cidr}
+    %{~ endif ~}
```

Comment:
`!= ""` is the way to test if the user didn't set it? (new to help, sorry for the noob question :D)

Can't we force the user to set this (if host_protection is enabled)?

Reply:
>!= "" is the way to test if the user didn't set it? (new to helm, sorry for the noob question :D)

I _think_ so, as the structure must be defined in values.yaml, as far as I know.

Reply:
Ah, this is actually Terraform template, not helm :)

Reply:
Hm, I wouldn't actually remove this check, as with Terraform, we are not able to validate, that this value is not empty and having `nodeCIDR: ""` in the result YAML pollutes the file, which may make it harder for debugging.